pkg/daemon/ceph/osd/encryption.go

```diff
@@ -32,8 +33,9 @@ import (
 )
 
 const (
-	cryptsetupBinary = "cryptsetup"
-	dmsetupBinary    = "dmsetup"
+	cryptsetupBinary   = "cryptsetup"
+	dmsetupBinary      = "dmsetup"
+	luksOpenCmdTimeOut = 30 * time.Second
```

Comment:
This looks quite long for a timeout, I would think that 10sec is more than enough already.